gulpfile.js

```diff
@@ -98,10 +96,7 @@ gulp.task('compile', function() {
   // change). Moreover, all *-internal.d.ts typings should not be exposed to 
   // developers as it denotes internally used types.
   if (declaration) {
-    workflow = workflow.pipe(filter([
-      'lib/**/*.js',
-      'lib/**/*.d.ts',
-      '!lib/**/*-internal.d.ts',
+    const TEMPORARY_TYPING_EXCLUDES = [
```

Comment:
Move this closer to the top of the file. Around where curatedTypings is. As this effort moves forward we will be adding things to curatedTypings and removing things from this list.